packages/sdk-ui-ts/src/services/ibc/ibcTokenMetadata.json

```diff
@@ -15,7 +15,8 @@
       "decimals": 18,
       "symbol": "INJ",
       "baseDenom": "INJ",
-      "isNative": false
+      "isNative": false,
```

Comment:
Maybe we can take this opportunity to remove "isNative" boolean flag, dont seem to see any `isNative: true` instance